python/cudf/cudf/core/buffer/buffer.py

```diff
@@ -277,20 +288,151 @@ def memoryview(
         )
         return memoryview(host_buf).toreadonly()
 
+    def __str__(self) -> str:
+        return (
+            f"<{self.__class__.__name__} size={format_bytes(self._size)} "
+            f"ptr={hex(self._ptr)} owner={self._owner!r}>"
+        )
+
+
+class Buffer(Serializable):
```

Comment:
Suggesting a hopefully non-controversial color for the bike shed:

`BufferOwner` -> `Buffer`
`Buffer` -> `BufferView`

Reply:
No strong opinion, the important thing is to make it clear that  `BufferOwner` is never to be used directly. 
 `Buffer` (current name) is the one that we should use throughout cudf.

Reply:
Since we are going to go round again in a final step of unification, let us punt a repeated renaming to there. But agreed, in that change we should have:

- `Buffer` (the "owning" object that contains the concrete devicebuffer/cupy array)
- `BufferView` (the "user-visible" object)

Reply:
We discussed offline and are in agreement that the renaming can happen in a follow-up.